catalog/app/containers/Auth/tests/__snapshots__/Auth.spec.js.snap

```diff
@@ -0,0 +1,381 @@
+      "is_staff": true,
+    },
+  },
+  "email": "admin@localhost",
+  "error": undefined,
+  "state": "REFRESHING",
+  "tokens": Object {
+    "access_token": "ACCESS1",
+    "expires_at": 1514707980,
+    "refresh_token": "REFRESH1",
+  },
+  "username": "admin",
+  "waiting": true,
+}
+`;
+
+exports[`Feature: containers/Auth/SignOut Background: Given minimal setup Scenario: Mounting the component when auth data is stale and then gets refreshed successfully Given storage has stale auth data When the component tree is mounted When auth refresh requests are resolved Then the user should be signed out 1`] = `
+<div>
+  <div
+    __name="Home"
+  />
+</div>
+`;
+
+exports[`Feature: containers/Auth/SignOut Background: Given minimal setup Scenario: Mounting the component when auth data is stale and then gets refreshed successfully Given storage has stale auth data When the component tree is mounted When auth refresh requests are resolved Then the user should be signed out 2`] = `
+Object {
+  "authenticated": false,
+  "domain": Object {
+    "state": "SIGNED_OUT",
+  },
+  "email": undefined,
+  "error": undefined,
+  "state": "SIGNED_OUT",
+  "tokens": Object {},
+  "username": undefined,
+  "waiting": false,
+}
+`;
+
+exports[`Feature: containers/Auth/SignOut Background: Given minimal setup Scenario: Mounting the component when auth data is stale and then gets refreshed with error Given storage has stale auth data When the component tree is mounted Then the user should see the activity indicator 1`] = `
+<div>
+  <div
+    __name="Working"
+  >
+    <span>
+      Signing out...
+    </span>
+  </div>
+</div>
+`;
+
+exports[`Feature: containers/Auth/SignOut Background: Given minimal setup Scenario: Mounting the component when auth data is stale and then gets refreshed with error Given storage has stale auth data When the component tree is mounted Then the user should see the activity indicator 2`] = `
+Object {
+  "authenticated": true,
+  "domain": Object {
+    "state": "REFRESHING",
+    "tokens": Object {
+      "access_token": "ACCESS1",
+      "expires_at": 1514707980,
+      "refresh_token": "REFRESH1",
+    },
+    "user": Object {
+      "current_user": "admin",
+      "email": "admin@localhost",
+      "is_active": true,
+      "is_staff": true,
+    },
+  },
+  "email": "admin@localhost",
+  "error": undefined,
+  "state": "REFRESHING",
+  "tokens": Object {
+    "access_token": "ACCESS1",
+    "expires_at": 1514707980,
+    "refresh_token": "REFRESH1",
+  },
+  "username": "admin",
+  "waiting": true,
+}
+`;
+
+exports[`Feature: containers/Auth/SignOut Background: Given minimal setup Scenario: Mounting the component when auth data is stale and then gets refreshed with error Given storage has stale auth data When the component tree is mounted When auth refresh requests are rejected Then the user should be signed out 1`] = `
+<div>
+  <div
+    __name="Home"
+  />
+  <div
+    __name="Notification"
+    message="Authentication lost"
```

Comment:
Can we have a better message? What has actually happend at this point? And should we just log people out?